### DISCUSSION TITLE
The .env file is always replaced when running dep deploy

### DISCUSSION THREAD

**Discussion by Akhmami** [deployphp/deployer#3946] [Help needed]
I'm using laravel recipe, by default I think the `.env` file will automatically be copied from the `.env.example` file and every time I run `dep deploy` the `.env` file is always replaced, how to handle this so that it only runs at the first time?

**Comment by antonmedv** [comment]
What Deployer version? 

You can also do this:

```php
task('deploy:env')->disable();
```

**Reply by Akhmami** [reply]
i am using the latest version, how do i do these in yaml?
